src/throttler-storage.interface.ts

```diff
@@ -3,14 +3,17 @@ export interface ThrottlerStorage {
    * The internal storage with all the request records.
    * The key is a hashed key based on the current context and IP.
    * The value of is an object contains the two keys:
-   *   - totalHits    (number): Amount of requests being done.
-   *   - timeToExpire (number): Amount of seconds until this key expires.
+   *   - totalHits: Amount of requests being done.
+   *   - expiresAt: Epoch milliseconds when this key expires.
    */
-  storage: Record<string, { totalHits: number; timeToExpire: number }>;
+  storage: Record<string, { totalHits: number; expiresAt: number }>;
```

Comment:
With how often we use the `{ totalHits: number, expiresAt: number }` type, should we make an interface for it? `ThrottlerData` or something like that?

Reply:
Ignore this. GitHub brought me to an old version of the PR